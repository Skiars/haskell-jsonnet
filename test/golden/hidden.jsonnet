{
  voice: {
    name :: "John",
    last_name: "Bon Jovi"
  },
  guitar :: {
    name: "Richie",
    last_name: "Sambora"
  }
}
