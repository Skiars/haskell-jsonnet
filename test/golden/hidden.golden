{"voice": {"last_name": "Bon Jovi"}}